public/ticketmanager.php

```diff
@@ -551,14 +693,14 @@
                     if ($prevOffset < 0) {
                         $prevOffset = 0;
                     }
-                    echo "<a href='/ticketmanager.php?g=$gameIDGiven&u=$assignedToUser&f=$gamesTableFlag&t=$ticketFilters'>First</a> - ";
-                    echo "<a href='/ticketmanager.php?o=$prevOffset&g=$gameIDGiven&u=$assignedToUser&f=$gamesTableFlag&t=$ticketFilters'>&lt; Previous $maxCount</a> - ";
+                    echo "<a href='$standardFilterURL" . "$ticketFilters'>First</a> - ";
+                    echo "<a href='/ticketmanager.php?o=$prevOffset&g=$gameIDGiven&u=$assignedToUser&p=$reportedByUser&r=$resolvedByUser&f=$gamesTableFlag&t=$ticketFilters'>&lt; Previous $maxCount</a> - ";
```

Comment:
Could use `standardFilterURL` here (and the next couple links) too. Just move the `o=` parameter to the end.

Reply:
Ended up creating a few template Url variables and eliminated almost all other calls now to standardize them as much as I could.